src/EditorFeatures/CSharpTest/ChangeSignature/RemoveParametersTests.cs

```diff
@@ -186,7 +187,7 @@ void B()
         [WpfFact, Trait(Traits.Feature, Traits.Features.ChangeSignature)]
         [WorkItem(1102830, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/1102830")]
         [WorkItem(784, "https://github.com/dotnet/roslyn/issues/784")]
-        public async Task RemoveParameters_ExtensionMethodInAnotherFile()
+        public async Task AddRemoveParameters_ExtensionMethodInAnotherFile()
```

Comment:
why did you cahnge the existing test?  can you undo this and add another tesT?

Reply:
Restored the preexisting test in the next review.